web/src/message_actions_popover.ts

```diff
@@ -20,27 +20,22 @@ import * as rows from "./rows.ts";
 import * as stream_popover from "./stream_popover.ts";
 import {parse_html} from "./ui_util.ts";
 import * as unread_ops from "./unread_ops.ts";
+import {the} from "./util.ts";
 
 let message_actions_popover_keyboard_toggle = false;
 
-function get_action_menu_menu_items() {
-    const $current_actions_popover_elem = $("[data-tippy-root] #message-actions-menu-dropdown");
-    if (!$current_actions_popover_elem) {
-        blueslip.error("Trying to get menu items when action popover is closed.");
-        return undefined;
-    }
```

Comment:
This error was not possible: `$(…)` always returns a jQuery object, which might be empty with `.length === 0` but cannot be falsy.

Reply:
The function is I think a duplicate of this, which looks like was also misconverted at some point:

```
function get_popover_items_for_instance(instance: tippy.Instance): JQuery | undefined {                 
    const $current_elem = $(instance.popper);                                                           
    const class_name = $current_elem.attr("class");                                                     
                                                                                                        
    if (!$current_elem) {                                                                               
        blueslip.error("Trying to get menu items when popover is closed.", {class_name});               
        return undefined;                                                                               
    }                                                                                                   
                                                                                                        
    return $current_elem.find("a, [tabindex='0']").filter(":visible");                                  
}                                                                                                       
```

I don't think these checks are very important; we've not seen them since the tippy migration.